packages/ws/src/types.ts

```diff
@@ -49,7 +49,7 @@ export interface ConnectionContext<TExtra> {
   // Whether the server has received connection init request from the client
   connectionInitReceived: boolean;
   // The "extra" variable
-  extra: TExtra;
+  extra?: TExtra;
```

Comment:
This is always defined (only it might just be `undefined` (https://github.com/hoangvvo/benzene/blob/main/packages/ws/src/handler.ts#L167). You can leave it as it.
```suggestion
  extra: TExtra;
```